eagle-security/eagle-security-maprfs-web/pom.xml

```diff
@@ -64,6 +64,16 @@
           <artifactId>eagle-security-common</artifactId>
           <version>${project.version}</version>
       </dependency>
+      <dependency>
+          <groupId>com.mapr.hadoop</groupId>
+          <artifactId>maprfs</artifactId>
+          <version>${maprfs.version}</version>
+      </dependency>
+      <dependency>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
+          <version>4.5.2</version>
```

Comment:
Please avoid declaring any dependency with explicit version but use property defined in root pom instead. Better way is to define in dependency-management firstly and include without version.